src/sth_database.js

```diff
@@ -5,7 +5,109 @@
 
   var mongoose = require('mongoose');
 
-  var sthConfig, sthLogger, sthHelper, connectionURL, eventSchema, aggregatedSchema;
+  var sthConfig, sthLogger, sthHelper, connectionURL, connection, eventSchema, aggregatedSchema;
+
+  /**
+   * Declares the Mongoose schemas.
+   */
+  function defineSchemas() {
+    switch (sthConfig.DATA_MODEL) {
+      case sthConfig.DATA_MODELS.COLLECTIONS_PER_SERVICE_PATH:
+        eventSchema = mongoose.Schema({
+          timestamp: Date,
+          entityId: String,
+          entityType: String,
+          attributeName: String,
+          attributeType: String,
+          attributeValue: Number
+        });
+        break;
+      case sthConfig.DATA_MODELS.COLLECTIONS_PER_ENTITY:
+        eventSchema = mongoose.Schema({
+          timestamp: Date,
+          attributeName: String,
+          attributeType: String,
+          attributeValue: Number
+        });
+        break;
+      case sthConfig.DATA_MODELS.COLLECTIONS_PER_ATTRIBUTE:
+        eventSchema = mongoose.Schema({
+          timestamp: Date,
+          attributeType: String,
```

Comment:
Really nice catch! ;) Yeah, that's a reminiscence from the past.. ;) Thank you very much for noticing!

Fixed in https://github.com/telefonicaid/IoT-STH/commit/afdb5eb5d0b51a6c0f4d4ee40b09777bc866fd3b